### PR TITLE
add dice roller package

### PR DESCRIPTION
**Pull request by micahgodbolt**
The intention is to bring this demo over separate from hello world. Here we can show how the same data can be rendered in different view frameworks, and aren't limited to keeping the demo incredibly sparse. Still a bit of a WIP, but wanted to port this over since i'll be removing the other views from the hello world repo.

### CODE REVIEW

File: multi-framework-diceroller/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# @fluid-example/multi-framework-diceroller
```
Comment (skylerjokiel) [inline multi-framework-diceroller/README.md:1]: This should eventually elaborate on how to set the different views. Don't block on it though.

File: multi-framework-diceroller/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@fluid-example/multi-framework-diceroller",
+    "version": "0.1.0",
+    "description": "Minimal Fluid Container & Data Object sample to implement a collaborative dice roller as a standalone app.",
+    "repository": "microsoft/FluidHelloWorld",
```
Comment (skylerjokiel) [inline multi-framework-diceroller/package.json:5]:
```suggestion
    "repository": "microsoft/FluidExamples",
```

File: multi-framework-diceroller/README.md (hunk 1 above)
Comment (micahgodbolt) [reply multi-framework-diceroller/README.md:1]: yup, plenty more to do in subsequent updates

**Review by skylerjokiel — APPROVED** [review]
🦒

**Comment by micahgodbolt** [comment]
pairs with https://github.com/microsoft/FluidHelloWorld/pull/163

**Pull request merged**